.github/dependabot.yml

```diff
@@ -12,18 +12,6 @@ updates:
         interval: weekly
       open-pull-requests-limit: 3
       rebase-strategy: disabled
```

Comment:
I can see this change is breaking unit tests for the module generator. Did you update it manually? I can see the `modules/compose` has been removed. If not, then I think there is a bug in the generator 🤔

Reply:
No, I didn't update it manually

Reply:
Ok, I'll investigate why and if found, will push a fix for the generator, thanks!